src/tad_dftd3/damping/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Collection of damping schemes.
+"""
+from .atm import *
+from .rational import *
```

Comment:
Can we make these import explicit?

Reply:
Why? The corresponding modules have `__all__` to avoid possible pollution by importing with `*`.

Reply:
The usage of ``__all__`` is currently inconsistent in the code base, not all modules define it. Also, there is a single function exported from ``.rational``, which does not require the usage of ``__all__``:

https://github.com/dftd3/tad-dftd3/blob/a4580ff0cec31a081ea276c8ebe0774dec5eb1ea/src/tad_dftd3/damping/rational.py#L9

Reply:
Changed for consistency.